### ISSUE TITLE
[Bug]: Cannot update

### ISSUE THREAD

**Issue by ghost** [ethical-haquer/Thor_GUI#18]
### What happened?

I tried to update Galaxy-Flasher but it didnt work, so I uninstalled it but Im getting this error message when using /.build.sh 
![Screenshot from 2024-08-05 18-27-48](https://github.com/user-attachments/assets/9bed8d56-6aec-4967-a225-1358d3d5e7b4)

### Galaxy Flasher version

Alpha v0.5.0

### Galaxy Flasher's log output

```shell
Running: pip install -r requirements.txt
Defaulting to user installation because normal site-packages is not writeable
Requirement already satisfied: PyGObject in /usr/lib/python3.11/site-packages (from -r requirements.txt (line 1)) (3.48.2)
Running: cp -r locales /app/locales/
Running: cp -r odin4-wrapper.sh /app/odin4-wrapper.sh
Running: cp -r usb.ids /app/usb.ids
cp: cannot stat 'usb.ids': No such file or directory
Error: module Galaxy-Flasher: Child process exited with code 1
```

**Comment by ethical-haquer** [comment]
@GoldenLife-01 
[There was a bug](https://github.com/ethical-haquer/Galaxy-Flasher/issues/15#issuecomment-2269884828) in the 5.1 release I first published. It has been fixed since then. If you're using GF, I would recommend re-downloading the release file, it should work now. Sorry for my delayed response!

**Comment by GoldenLife-01** [comment]
I fixed it by copying 5.0 files into the 5.1 directory and keeping the 5.1 files if there were any conflicts, and I compiled it like usual and it worked, I don't know if it was the fixed version, but things work for me now > @GoldenLife-01 
> [There was a bug](https://github.com/ethical-haquer/Galaxy-Flasher/issues/15#issuecomment-2269884828) in the 5.1 release file I first published. It has been fixed since then. If you're using GF, I would recommend re-downloading the release file, it should work now. Sorry for my delayed response!
>

**Comment by ethical-haquer** [comment]
> I fixed it by copying 5.0 files into the 5.1 directory and keeping the 5.1 files if there were any conflicts, and I compiled it like usual and it worked, I don't know if it was the fixed version, but things work for me now
> 
> > @GoldenLife-01
> > [There was a bug](https://github.com/ethical-haquer/Galaxy-Flasher/issues/15#issuecomment-2269884828) in the 5.1 release file I first published. It has been fixed since then. If you're using GF, I would recommend re-downloading the release file, it should work now. Sorry for my delayed response!

Yeah, you basically would have copied the usb.ids file then, which would fix the problem. Nice work!
